src/api/controller/front.js

```diff
@@ -343,6 +348,15 @@ app.use(
     }),
 );
 
+for (let theme of getAvailableTheme()) {
+    app.use(
+        mount(
+            `/themes/${theme}/`,
+            serve(path.resolve(__dirname, `../../themes/${theme}`)),
+        ),
```

Comment:
It seems that when you mount a theme, you can't specialize any files of the theme, or simply add files (from `custom` directory) any more.
Am I wrong?

Reply:
custom directory still work, but this mount the theme directory to get acces to theme files